screen-orientation/orientation-reading.html

```diff
@@ -40,21 +40,21 @@
           secondaryOrientation2: "portrait-secondary"
         };
   await screen.orientation.lock(orientations.secondaryOrientation1);
-  assert_true(
+  assert_equals(
```

Comment:
this should be:
```
assert_equals(screen.orientation.angle, 180, "Secondary orientation 1 angle must be 180") 
```
i.e., assert_equals takes 3 arguments.

Reply:
Sorry!! Wasn't thinking properly!

Reply:
No problem. What reviews are for 🥳